index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <mate charset="utf-8">
+    <title>Linkbrary</title>
+    <link 
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="reset.css">
+  </head>
+    <body>
+      <header>
+        <nav>
+          <a href="index.html">
+            <img src="images/logo.svg" alt="홈으로 연결된 Linkbrary 로고">
+          </a>
+          <a class="cta cta-short" href="signin.html">
+            <span>로그인</span>
+          </a>
+        </nav>
+        <div class="hero-header">
+          <h1 class="slogan">
+            <span class="slogan-gradient background-clip-text">세상의 모든 정보</span>를
+            <br>
+            쉽게 저장하고 관리해 보세요
+          </h1>
+          <a class="cta cta-long" href="signup.html">
+            <span>링크 추가하기</span>
+          </a>
+          <img src="images/hero.png" class="hero-image" alt="Linkbrary 서비스 소개">
+        </div>
+      </header>
+      <article>
+        <section>
+          <h2 class="title">
+            <span class="title-1-gradient background-clip-text">원하는 링크</span>를
+            <br>
+            저장하세요
+          </h2>
+          <p class="description">
+            나중에 읽고 싶은 글, 다시 보고 싶은 영상,<br>
+            사고 싶은 옷, 기억하고 싶은 모든 것을<br>
+            한 공간에 저장하세요.
+          </p>
+          <img src="images/image1.png" class="content-image" alt="링크의 내용이 담긴 카드들">
+        </section>
+        <section>
+          <h2 class="title">
+            링크를 폴더로<br>
+            <span class="title-2-gradient background-clip-text">관리</span>하세요
+          </h2>
+          <p class="description">
+            나만의 폴더를 무제한으로 만들고<br>
+            다양하게 활용할 수 있습니다.
+          </p>
+          <img src="images/image2.png" class="content-image" alt="폴더 이름 변경 기능">
+        </section>
+        <section>
+          <h2 class="title">
+            저장한 링크를<br>
+            <span class="title-3-gradient background-clip-text">공유</span>해 보세요
+          </h2>
+          <p class="description">
+            여러 링크를 폴더에 담고 공유할 수 있습니다.<br>
+            가족, 친구, 동료들에게 쉽고 빠르게 링크를<br>
+            공유해 보세요
+          </p>
+          <img src="images/image3.png" class="content-image" alt="폴더 공유 기능">
+        </section>
+        <section>
+          <h2 class="title">
+            저장한 링크를<br>
+            <span class="title-4-gradient background-clip-text">검색</span>해 보세요
+          </h2>
+          <p class="description">
+            중요한 정보들을 검색으로 쉽게 찾아보세요.
+          </p>
+          <img src="images/image4.png" class="content-image" alt="링크 검색 기능">
+        </section>
+      </article>
+      <footer>
+        <div class="footer-box">
+          <span class="copyright">@codeit - 2023</span>
+          <div class="footer-links">
+            <a class="footer-link" href="privacy.html">Privacy Policy</a>
```

Comment:
```suggestion
            <a class="footer-link" target="_blank" href="privacy.html">Privacy Policy</a>
```
위처럼 target="_blank"를 추가해주시면 새로운 탭으로 열리는데 주로 푸터에 들어가는 약관에 대한 링크는 새로운탭에 여는걸 추천드려요 혹시나 궁금하면 네이버에 들어가셔서 확인해보셔도 좋을것 같아요:)